ccloud/docs/beginner-cloud.rst

```diff
@@ -390,7 +384,7 @@ Run a Java producer before and after configuring the ACLs
          User:104349      | ALLOW      | CREATE    | TOPIC    | demo-topic-1 | LITERAL
          User:104349      | ALLOW      | WRITE     | TOPIC    | demo-topic-1 | LITERAL
 
-#. Run the Java producer to ``demo-topic-1`` after configuring the ACLs:
+#. Run the Java producer to ``demo-topic-1`` after configuring the ACLs (expected to PASS):
 
```

Comment:
Should "PASS" be in lowercase here? We have "fail" in lowercase in line 335.

Reply:
Good catch, fixed.